azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+# https://docs.microsoft.com/azure/devops/pipelines/ecosystems/python
+pool:
+  vmImage: "ubuntu-16.04"
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.6'
+
+- script: python -m pip install --upgrade pip setuptools wheel
+  displayName: 'Install tools'
+
+- script: pip install -r dev-requirements.txt
+  displayName: 'Install dev requirements'
+
+- script: pip install -e .
+  displayName: 'Install r2d'
+
+# venv conda 
+- script: pytest --durations 10 --cov repo2docker -vs tests/unit
+  displayName: 'Run pytest'
```

Comment:
@willingc if I wanted to add more tests like we have for travis

<img width="831" alt="Screenshot 2019-09-08 at 11 37 31" src="https://user-images.githubusercontent.com/1448859/64486407-16d61180-d22d-11e9-9fe7-a5e047d9dc9c.png">

then the thing to do is add new versions of these last two lines here?

Reply:
Yes. This was just to get the bare tooling in before I leave. Next steps would be to matrix the tests and add the operating systems.